x-pack/plugins/security_solution/server/lib/entity_analytics/utils/create_datastream.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 // This file is a copy of x-pack/plugins/alerting/server/alerts_service/lib/create_concrete_write_index.ts
-// original function create index instead of datastream, and their have plan to use datastream in the future
+// original function create index instead of datastream, and they have plans to use datastream in the future
 // so we probably should remove this file and use the original when datastream will be supported
```

Comment:
If I were touching this at all, I would probably go a little further here: 
```suggestion
// The original function created an index, while here we create a datastream. If and when responseOps develops first-party code to work with datastreams (https://github.com/elastic/kibana/issues/140403), this file should be removed.
```

Reply:
Beautiful! Gives me a chance to kick off the buildkite again anyway now that I've got the labels I want on the PR 😄